### PR TITLE
Support different behat tags for deploy, pr, cron, local

### PR DESCRIPTION
**Pull request by aweingarten**

### CODE REVIEW

**Comment by grasmash** [comment]
Looks good. Can you add some documentation about this feature? It should be mentioned in ci.md.

**Comment by aweingarten** [comment]
@grasmash  I just added documentation.  

This code has been merged to CB.  Going to test there for the 2-3 days before giving you the green light to merge.  It looks like the PR builds are working great.  Want to see a cron run and a deploy.

File: phing/build.yml [hunk 1]
```diff
@@ -17,6 +17,10 @@ behat:
     - ${docroot}/profiles
     - ${repo.root}/tests/behat
   tags: '~ajax&&~experimental&&~lightningextension'
+  tags-by-build-type:
```
Comment (grasmash) [inline phing/build.yml]: Looking at this with fresh eyes, it seems like we should allow for more than dynamic behat tests. We could, for instance, allow a user to just define a bash script that gets called.

**Comment by grasmash** [comment]
Closings pull request as per the latest comment. I'd like to support nightly builds, but I think that we need to broaden the scope beyond Behat tags.

**Pull request closed without merging**